generators/Readme.md

```diff
@@ -0,0 +1,302 @@
+# Generators
+
+- Owner: @timsuchanek
+- Stakeholders: @schickling
+- State:
+  - Spec: In Progress 🚧
+  - Implementation: In Progress 🚧
+
+An important mechanism in Prisma 2 is the generation of code based on the Prisma schema.
+This spec describes, how the interface for such code generation in the individual languages like JavaScript and TypeScript looks like, and which protocol is used under the hood to achieve that interface.
+
+---
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Generator architecture](#generator-architecture)
+- [Information passed into a generator](#information-passed-into-a-generator)
+- [Interface for a generator](#interface-for-a-generator)
+  - [1. Create an executable file](#1-create-an-executable-file)
+  - [2. Point to the generator file in the `schema.prisma`](#2-point-to-the-generator-file-in-the-schemaprisma)
+  - [3. Run `prisma2 generate`](#3-run-prisma2-generate)
+- [Interface between Generator SDK and each language helper](#interface-between-generator-sdk-and-each-language-helper)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Generator architecture
+
+A generator is a piece of software, which gets metadata of the Prisma Schema like the models passed in by the Generator SDK and generates code based on it.
+The generator is responsible for saving that code to the filesystem.
+
+```bash
+┌────────────────┐
+│ Generation SDK │
+└─┬──────────────┘
+  │
+  │ passes schema meta data
+  │
+  ▼
+┌─────────────────┐
+│    Generator    │
+└─┬───────────────┘
+  │
+  │ generates code
+  │
+  ▼
+┌─────────────────┐
+│   Filesystem    │
+└─────────────────┘
+```
+
+## Information passed into a generator
+
+A generator gets passed in a JSON blob including the following information:
+
+1. Everything which is declared in the `schema.prisma` file:
+
+- Models
```

Comment:
```suggestion
   - Models
```
to get the unordered list as a sub thing of the numbered list (also for all the other items of course)